.github/CODEOWNERS

```
@@ -0,0 +1,3 @@
+# Team Sol is taking ownership over delayed job infrastructure.
+# This will notify us of any requested changes to anything
+* @cultureamp/team-sol
```

at least on my page it is showing that the codeowners file contains an error

```
Unknown owner on line 3: make sure the team @cultureamp/team-sol exists and has write access to the repository
* @cultureamp/team-sol
```

---

I think it just needs to be updated to this (I've added sol as write access)
```suggestion
* @cultureamp/sol
```